Deprecation and Distributed keyword removal

CSSFrancis opened the pull request:
### Description of the change
Remove the distributed keyword for `.mrc` see #400 

### Progress of the PR
- [ ] Change implemented (can be split into several points),
- [ ] update docstring (if appropriate),
- [ ] update user guide (if appropriate),
- [ ] add a changelog entry in the `upcoming_changes` folder (see [`upcoming_changes/README.rst`](https://github.com/hyperspy/rosettasciio/blob/main/upcoming_changes/README.rst)),
- [ ] Check formatting of the changelog entry (and eventual user guide changes) in the `docs/readthedocs.org:rosettasciio` build of this PR (link in github checks)
- [ ] add tests,
- [ ] ready for review.

CSSFrancis commented:
@ericpre I'm not sure if this is exactly the right way to handle this removal but I adjusted the deprecation code in pyxem to handle this.

ericpre reviewed (commented):
Something breaks the test suite, it is not obvious to see what it is from the reading the code.

Add test for the functions in the `_deprecated` module?

CSSFrancis commented:
pre-commit.ci autofix

CSSFrancis commented:
> Something breaks the test suite, it is not obvious to see what it is from the reading the code.
> 
> Add test for the functions in the `_deprecated` module?

Ah sorry, I deleted the distributed keyword in the doc string but didn't remove the string formatting.  Should be good now.  I added some tests fro the deprecations as well.
